bundles/nl.asml.matala.product/src/nl/asml/matala/product/Product.xtext

```diff
@@ -121,7 +121,7 @@ SymbConstraint:
 
 DataAssertions: {DataAssertions}
     'assertions' name = ID '{' 
-        constr += DataCheckItems*
+        constr += DataAssertionItem*
```

Comment:
Please use separate branches (branched off from from main) for separate features. Currently this pull request suddenly contains assert language changes, which is not the intention I guess.